pages/residents/[id]/index.tsx

```diff
@@ -486,22 +490,25 @@ const ResidentPage = ({ resident }: Props): React.ReactElement => {
         }
       >
         <>
-          {cases && (
-            <>
-              <CaseNoteGrid
-                cases={cases}
-                resident={resident}
-                totalCount={totalCount}
-              />
-              {canManage && (
-                <Link href={`/residents/${resident.id}/case-notes`}>
-                  <a className="lbh-link lbh-link--muted lbh-body-xs govuk-!-margin-top-2">
-                    See all {totalCount} case notes & records
-                  </a>
-                </Link>
-              )}
-            </>
-          )}
+          {cases &&
```

Comment:
As there is quite a bit of code in the frontend pages here, it would be good to see some cypress tests covering this, or some unit/acceptance tests if you prefer.